tests/tools/oesign-engine/test-enclave/CMakeLists.txt

```diff
@@ -143,3 +143,11 @@ add_test(
 
```

Comment:
I don't think this test runs or builds given the changes in master (oesign-engine has been removed as a folder and is now part of oesign/test-engine) See comments about rebasing and dropping the oesign changes to simplify.

Reply:
Update code on latest master to get refactor change